index.html

```diff
@@ -1,192 +0,0 @@
-<!DOCTYPE HTML>
-<html>
-
-<head>
-  <title>Mockup Home</title>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  <link href="./index.css" rel="stylesheet" />
-</head>
-
-<body>
-  <!-- header section -->
-  <section id="header">
-    <h1><a href="./index.html">Home Page</a></h1>
-    <h4>The Page For The Home.</h4>
-  </section>
-  <hr>
-  <hr>
-  <section id="features"></section>
-
-  <section id="main">
-    <!-- three photo section below header with button -->
-    <p class="container">
-    <h2>TAKE A LOOK AT THIS AMAZING HOME PAGE!</h2>
-    <img src="./images/goku-img.jpg" alt=" " />
-    <img src="./images/gokuss.jpg" alt=" " />
-    <img src="./images/ssj3.jpg" alt=" " />
-
-    <button><a href="./learn-more.html" class='nav-button'>Tell Me More</a></button>
-    </p>

```

Comment:
Breaking up your page a bit more will make styling the layout SOO much easier. For instance, this `three photo section` could be wrapped in it's own containing `<div>`, and then that new `<div>` could be set to `display: flex; justify-content: center;`. This would get everything in this section centered on the page.